src/components/NavBar.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import '../styles/NavBar.css'
 
 const NavBar = ({goHome}) => {
```

Comment:
As a minor formatting remark - I see you using spaces between the curly brackets when destructuring elsewhere, and this is a common convention. So change `{goHome}` to `{ goHome }`.